interfaces/cython/cantera/onedim.pyx

```diff
@@ -467,17 +467,52 @@ cdef class _FlowBase(Domain1D):
         def __set__(self, do_radiation):
             self.flow.enableRadiation(<cbool>do_radiation)
 
+    def set_freeFlow(self):
+        self.flow.setFreeFlow()
+
+    def set_axisymmetricFlow(self):
+        self.flow.setAxisymmetricFlow()
+
```

Comment:
Please add docstrings for these two methods. Also, I think the preferred Python method names would be `set_free_flow` and `set_axisymmetric_flow`.